tests/test_shift_intensityd.py

```diff
@@ -14,16 +14,17 @@
 import numpy as np
 
 from monai.transforms import IntensityStatsd, ShiftIntensityd
-from tests.utils import NumpyImageTestCase2D
+from tests.utils import TEST_NDARRAYS, NumpyImageTestCase2D, assert_allclose
 
 
 class TestShiftIntensityd(NumpyImageTestCase2D):
     def test_value(self):
         key = "img"
-        shifter = ShiftIntensityd(keys=[key], offset=1.0)
-        result = shifter({key: self.imt})
-        expected = self.imt + 1.0
-        np.testing.assert_allclose(result[key], expected)
+        for p in TEST_NDARRAYS:
```

Comment:
Could you please help add some test cases for Tensor data?

Thanks.

Reply:
the for loop across `TEST_NDARRAYS` will test `np.ndarray`, `torch.Tensor` (cpu) and `torch.Tensor` (cuda, if available).

Reply:
Got it, I thought the `TEST_NDARRAYS` here means `np.ndarray`...
Thansk.